_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -18,17 +20,18 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to any [keyboard event][]: 
- - where the attribute `key` is a [printable character][] key; and
- - the method `getModifierState` returns `false`; and
- - is [dispatched][] to an [event target][] within a [HTML document][].
+The rule applies to any [keyboard event][] for which all is true: 
+- the event's attribute `key` is a [printable character][] key; and
+- the event's method `getModifierState` returns `false`; and
+- the event is [dispatched][] to an [event target][] within a [HTML document][];
+- the event causes [changes in content] of the [HTML document][].
 
 ## Expectation
 
 For each test target at least one of the following is true:
- - (**Disable**:) There is at least one [set of clearly labelled instruments][] to [block the event][blocked event]; or
- - (**Remap**:) There is at least one [set of clearly labelled instruments][] to [block the event][blocked event] when the [keyboard event][] method `getModifierState` doesn't return `true`; or
- - (**Focus**:) If the [event target][] doesn't have [focus][] the [event is blocked][blocked event].
+ - **Disable**: there is at least one [set of clearly labelled instruments][] to [block events][blocked event] that use the [same key][same key events] of the test target; or
+ - **Remap**: there is at least one [set of clearly labelled instruments][] to [block events][blocked event] that use the [same key][same key events] of the test target, unless the non-test target event `getModifierState` method returns `true` for any of the [valid modifier keys][]; or
+ - **Focus**: if the [event target][] doesn't have [focus][] the [event is blocked][blocked event].
```

Comment:
Should that one use "makes no change in content"?
Same thing, again, but different feeling for the reader…

Not sure.

Reply:
Glad you made this comment @Jym77 
It made me realize that the applicability cannot require that the event causes any change in the content of the document. Otherwise, it could never meet this part of the expectation. 

But I also prefer a formulation like "does not change the content" because it's more passive (i.e. the user is not required to do anything), versus the active "block the event by activating an instrument".